run_project.md

```diff
@@ -0,0 +1,44 @@
+# How to run the project after installing everything
```

Comment:
Because we use Docker, which is sort of virtual machines, we only need to install Docker in workstations.
It's unclear what everything means here